bot/cogs/filtering.py

```diff
@@ -126,6 +136,53 @@ async def on_message_edit(self, before: Message, after: Message) -> None:
             delta = relativedelta(after.edited_at, before.edited_at).microseconds
         await self._filter_message(after, delta)
 
+    @staticmethod
+    def get_name_matches(name: str) -> List[re.Match]:
+        """Check bad words from passed string (name). Return list of matches."""
+        matches = []
+        for pattern in WATCHLIST_PATTERNS:
+            match = pattern.search(name)
+            if match:
+                matches.append(match)
+        return matches
+
+    async def check_send_alert(self, member: Member) -> bool:
+        """When there is less than 3 days after last alert, return `False`, otherwise `True`."""
+        last_alert = await self.name_alerts.get(member.id)
+        if last_alert:
+            last_alert = datetime.fromtimestamp(last_alert)
+            if datetime.now() - timedelta(days=DAYS_BETWEEN_ALERTS) < last_alert:
+                return False
+
+        return True
+
+    async def check_is_bad_words_in_name(self, member: Member) -> None:
```

Comment:
Sorry to nitpick, but this is awkward to read since it isn't grammatically correct (should be `are` since `words` is plural). I suggest just dropping the `is` entirely: `check_bad_words_in_name`.